RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -596,6 +596,11 @@ void consumeStuff()
 				return;
 			}
 		}
+	
+		if (spleen_left() >= 4 && !isActuallyEd())
+		{
+			auto_chewAdventures();
```

Comment:
Consistency: missing a return here (see the above uses of `auto_autoConsumeOne` for examples).
Yes this is the last thing in the function right now however if someone else adds something to this function it will no longer be.

Reply:
resolved by @quarklikeadork